src/qt/walletmodel.cpp

```diff
@@ -667,8 +669,9 @@ bool WalletModel::bumpFee(uint256 hash)
 {
     std::unique_ptr<CFeeBumper> feeBump;
     {
+        CCoinControl no_coin_control;
         LOCK2(cs_main, wallet->cs_wallet);
-        feeBump.reset(new CFeeBumper(wallet, hash, nTxConfirmTarget, false, 0, true));
+        feeBump.reset(new CFeeBumper(wallet, hash, no_coin_control, 0));
```

Comment:
Good catch.  Will fix. I think I just assumed how the code worked and didn't even look at what I was replacing.